### ISSUE TITLE
Unable to move exercism file to bin folder 

### ISSUE THREAD

**Issue by preston24** [exercism/exercism#4222]
When running this command "mv ~/Downloads/exercism ~/bin" it tells me No such file or directory.

**Comment by NobbZ** [comment]
Do you have created the `~/bin` folder first? And have you extracted the tarball into your `~/Downloads` folder?

**Comment by preston24** [comment]
Yes I have `bin` folder and yes I extracted the tarball.

**Comment by NobbZ** [comment]
Then please do the following and report back with the output:

```shell
ls -lah ~/bin
ls -lah ~/Downloads/exercism
```

**Comment by preston24** [comment]
`~ » ls -lah ~/bin                                  
total 0
drwxr-xr-x   2 prestonbayle  staff    64B Aug 11 11:21 .
drwxr-xr-x+ 33 prestonbayle  staff   1.0K Aug 11 16:54 ..`

`~ » ls -lah ~/Downloads/exercism                     
ls: /Users/prestonbayle/Downloads/exercism: No such file or directory`

**Comment by preston24** [comment]
This is what is in my `Downloads`. After I extracted the tarball it gave me this folder `exercism-mac-64bit` which then has two things in it, `exercism` and another folder called `shell`  which contains `exercism_completion.bash`, `exercism_completion.zsh` and `README.md`.

**Comment by NobbZ** [comment]
Then it seems as if the tutorial hasn't updated after package structure changed.

Please `mv ~/exerercism-mac-64bit/exerercism ~/bin/exerercism`.

**Comment by NobbZ** [comment]
Yeah, that's because my autocorrect made stupid stuff, just replace garbage by exercism in the command I wrote.

**Comment by preston24** [comment]
Ha. I didn't even notice that. Unfortunately fixing the spelling errors didn't work either. Still getting the same error.
`mv: rename /Users/prestonbayle/exercism-mac-64bit/exercism to /Users/prestonbayle/bin/exercism: No such file or directory`
